profiles/VP_ANDROID_15.json

```diff
@@ -0,0 +1,205 @@
+{
+    "$schema": "https://schema.khronos.org/vulkan/profiles-0.8.1-237.json#",
+    "capabilities": {
+        "MUST": {
+            "extensions": {
+                "VK_KHR_maintenance5": 1,
+                "VK_KHR_shader_float16_int8": 1,
+                "VK_KHR_16bit_storage": 1,
+                "VK_KHR_vertex_attribute_divisor": 1,
```

Comment:
`VK_KHR_vertex_attribute_divisor` doesn't exist, do you mean `VK_EXT_vertex_attribute_divisor`?

Reply:
The KHR version of the extension is up for board ratification.  IIUC, it was promoted priort to being included in the Roadmap 2024 profile.

Reply:
What are others doing about promoted extensions?  Should we use either-or requirements, because both the EXT and KHR extensions are both fine.

Reply:
I guess it depends, who are the others. 
Considering this caveat, I would say, typically, if there is not change in functionnalities, the oldest extension is used.